From: "xypron.glpk@gmx.de" <xypron.glpk@gmx.de>
Date: Sun, 31 Jul 2016 12:07:37 +0200
Subject: net: qlcnic: avoid superfluous assignement
Patch-mainline: v4.8-rc1
Git-commit: 43dcff349f091b6a2f5f491c35d89b173df41acf
References: bsc#1019717 FATE#321698

Assigning NULL to parmeter dcb has no effect outside of the
inlined function.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/qlogic/qlcnic/qlcnic_dcb.h |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_dcb.h
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_dcb.h
@@ -45,7 +45,6 @@ struct qlcnic_dcb {
 static inline void qlcnic_clear_dcb_ops(struct qlcnic_dcb *dcb)
 {
 	kfree(dcb);
-	dcb = NULL;
 }
 
 static inline int qlcnic_dcb_get_hw_capability(struct qlcnic_dcb *dcb)
